docs/README.md

```diff
@@ -1,188 +1,369 @@
-# TileDB for summary statistics data
+# Usage documentation
+
+- [Usage documentation](#usage-documentation)
+- [1. Ingestion](#1-ingestion)
+  - [1.1 Input](#11-input)
+    - [1.1.1 Summary statistics preparation](#111-summary-statistics-preparation)
+    - [1.1.2 Metadata preparation](#112-metadata-preparation)
+  - [1.2 Command](#12-command)
+- [2. Export](#2-export)
+  - [2.1 Input](#21-input)
+    - [2.1.1 Query file](#211-query-file)
+  - [2.2 Command](#22-command)
+  - [2.3 Output](#23-output)
+    - [2.3.1 SNP, region, all selection](#231-snp-region-all-selection)
+    - [2.3.2 Locusbreaker](#232-locusbreaker)
+    - [2.3.1 Metadata](#231-metadata)
+- [3. Meta-query](#3-meta-query)
+  - [3.1 Input](#31-input)
+  - [3.2 Command](#32-command)
+  - [3.3 Output](#33-output)
+- [4. Deployment](#4-deployment)
```

Comment:
I suggest to move deployment to a completly separated document. I will take care of it

Reply:
ok you do here or on a different branch?